.github/workflows/workflow.yml

```diff
@@ -13,7 +13,7 @@ jobs:
     - name: Set up Go
       uses: actions/setup-go@v3
       with:
-        go-version: 1.19
```

Comment:
Can we use `matrix` so that we can upgrade later in a single place?

Reference: https://github.com/notaryproject/notation/blob/e47cf1253dcf6af1c7c9eb0688180a523a324d98/.github/workflows/release-github.yml#L14

Reply:
Updated.